<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-R3MDHTL" failures="3" tests="4" name="Default test(failed)(failed)" time="0.107" errors="0" timestamp="2022-07-26T20:34:43 IST">
  <testcase classname="testNGfailed.FailTC" name="a" time="0.017">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNGfailed.FailTC.a(FailTC.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- a -->
  <testcase classname="testNGfailed.FailTC" name="d" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNGfailed.FailTC.d(FailTC.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- d -->
  <testcase classname="testNGfailed.FailTC" name="f" time="0.003">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNGfailed.FailTC.f(FailTC.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase classname="testNGfailed.FailTC" name="b" time="0.003"/>
  <testcase classname="testNGfailed.FailTC" name="g" time="0.0">
    <ignored/>
  </testcase> <!-- g -->
  <testcase classname="testNGfailed.FailTC" name="e" time="0.0">
    <ignored/>
  </testcase> <!-- e -->
  <testcase classname="testNGfailed.FailTC" name="h" time="0.0">
    <ignored/>
  </testcase> <!-- h -->
  <testcase classname="testNGfailed.FailTC" name="c" time="0.0">
    <ignored/>
  </testcase> <!-- c -->
</testsuite> <!-- Default test(failed)(failed) -->
